rita_exit/src/operator_update/update_loop.rs

```diff
@@ -13,24 +27,35 @@ pub fn start_operator_update_loop() {
     thread::spawn(move || {
         // this will always be an error, so it's really just a loop statement
         // with some fancy destructuring
+
         while let Err(e) = {
-            thread::spawn(move || loop {
-                let start = Instant::now();
-                trace!("exit Update loop tick!");
-
-                let runner = AsyncSystem::new();
-                runner.block_on(async move {
-                    // Check in with Operator
-                    operator_update(rita_started).await;
-                });
-
-                info!(
-                    "exit Operator Update loop completed in {}s {}ms",
-                    start.elapsed().as_secs(),
-                    start.elapsed().subsec_millis()
-                );
+            thread::spawn(move || {
+                // No need to query database every 60secs to get the registered list, keep a cache and
```

Comment:
why cache versus setting the checkin time to every 10 minutes? 

On the ops side people may be a little confused to not see the output of their register command for 10 minutes. 

Considering that the exits have hundreds of routers making db requests pretty often,  I think we shouldn't bother with this cache until we can confirm we really need it. Any caching or optimization really needs to happen on the router status checkin side.

Reply:
Ok I removed the cache